eqt/ui/UIFormWidget.py

```diff
@@ -161,6 +164,28 @@ def _addWidget(self, name, qwidget, qlabel=None):
 
         formLayout.setWidget(widgetno, field_form_role, qwidget)
         self.num_widgets += 1
+        self.populate_default_widget_states_dictionary(name, qlabel)
+
+    def populate_default_widget_states_dictionary(self, name, qlabel):
+        '''
+        Creates an attribute dictionary of default widget states. The entries are in the
+        format: {'value': str | bool | int, 'enabled': bool, 'visible': bool}.
+        This can be used to restore the default states of the widgets invoking `applyWidgetStates`.
+        '''
+        if not hasattr(self, 'default_widget_states'):
+            self.default_widget_states = {}
+        # add the default state of the qwidget
+        self.default_widget_states[f'{name}_field'] = self.getWidgetState(name, 'field')
+        # add the default state of the qlabel
+        if qlabel is not None:
+            self.default_widget_states[f'{name}_label'] = self.getWidgetState(name, 'label')
+
+    def make_default_widget_states_visible(self):
```

Comment:
The method name doesn't seem to be correct. The method does not set the widgets visible.
```suggestion
    def set_default_widget_states_visible_true(self):
```

Reply:
Fair enough, I changed it thank you